ArduCopter/mode_systemid.cpp

```diff
@@ -369,101 +366,25 @@ void ModeSystemId::run()
         
     } else {
 
-        // process pilot inputs unless we are in radio failsafe
-        // this was split into two parts to allow pilot input to accomplish closed loop sweeps
-        if (!copter.failsafe.radio) {
-
-            // Set pilot's roll and pitch input to zero
-            loiter_nav->set_pilot_desired_acceleration(0.0f, 0.0f);
```

Comment:
So we don't have to update the pilot's desired accelerations?  I thought I would need to set this to zero